spec/features/supply_spec.rb

```diff
@@ -0,0 +1,125 @@
+    fill_in t("#{base}.name"),         with: supply.name
+    fill_in t("#{base}.description"),  with: supply.description
+    fill_in t("#{base}.due_date"),     with: supply.due_date                      
+    fill_in t("#{base}.location"),     with: supply.location
+    
+    click_button  t("helpers.submit.create", model: t_supply)
+    supply
+  end
+
+  def visit_add_supply(user)
+    visit(circle_path(circle, as: user))
+    find('.button-super', text: /#{t("sidebar.circle_add_menu.add")}/).click
+    click_on t_supply
+  end
+
+  before do
+    create(:circle_role_admin, circle: circle, user: user_1) 
+    create(:circle_role_volunteer, circle: circle, user: user_2)
+
+    public_group.roles.member.create              user: user_1
+    public_group.roles.member.create              user: user_2
+
+    private_group.roles.member.create             user: user_1
+  end
+
+  context 'add supply button' do
+    let(:add_text) { t("sidebar.circle_add_menu.add") }
+
+    it 'is visible to Admin' do
+      visit(circle_path(circle, as: user_1))
+      
+      expect(find('.button-super', text: /#{add_text}/)).to_not eq(nil)
+      find('.button-super', text: /#{add_text}/).click
+      expect(page).to have_link(t_supply)
+    end
+
+    it 'not visible to Volunteer' do
+      visit(circle_path(circle, as: user_2))
+      expect(page).to_not have_selector('.button-super', text: /#{add_text}/)
+    end
+  end
+
+  context 'an admin' do
+    it "adds a supply" do
+      visit_add_supply(user_1)
+      s = create_supply(FactoryGirl.create(:supply))
+
+      expect(page).to have_content(t("flash.actions.create.notice", resource_name: t_supply))
+      expect(page).to have_content(s.name)
+      expect(page).to have_content(s.description)
+
+      expect(page).to have_content("#{t("activerecord.attributes.supply.due_date")} #{s.due_date.strftime("%A %-d %B %Y")}")
+      expect(page).to have_content(s.location)
+    end
+
+    it "edits a supply" do
+      visit_add_supply(user_1)
+      s = create_supply(FactoryGirl.create(:supply))
+      find('.button-super', text: /#{t("helpers.edit", model: t_supply)}/).click
+      click_on  t("circle.supplies.edit_actions.edit_button_dropdown")
+      s.name =  "Edited #{s.name}"
+      fill_in   t("activerecord.attributes.supply.name"), with: s.name
+  
+      click_on  t("helpers.submit.update", model: t_supply)
+      expect(page).to have_content t("flash.updated", name: t_supply)
+      expect(page).to have_content s.name
+    end
+
+    it "has a supply" do
+      visit_add_supply(user_1)
+      s = create_supply(FactoryGirl.create(:supply))
+
+      click_on  t("helpers.volunteer", model: t_supply)
+      expect(page).to have_content t("helpers.decline", model: t_supply)
+      expect(page).to have_content(t("suppliers.auto_comment.user_assigned", 
+                                     user: user_1.name, 
+                                     date: s.due_date.strftime("%B %-d, %Y")))
```

Comment:
Use I18n.l(Date.today, format: :long) to match the current date and format that the commenter code uses. Same problem in another test.